app/feedme/html_parser.py

```
@@ -0,0 +1,965 @@
+        
+        for fmt in formats:
+            try:
+                return datetime.strptime(timestamp_str, fmt)
+            except ValueError:
+                continue
+        
+        # Try parsing with dateutil if available
+        try:
+            from dateutil import parser
+            return parser.parse(timestamp_str)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)